agml/_internal/preprocess.py

```diff
@@ -816,9 +817,51 @@ def plant_doc_detection(self, dataset_name):
             extract_num_from_imgid=False
         )
 
+    def wheat_head_counting(self, dataset_name):
+        label2id = {"Wheat Head": 1}
 
+        dataset_dir = os.path.join(self.data_original_dir, dataset_name)
+        anno_files = [os.path.join(dataset_dir, 'competition_train.csv'), os.path.join(dataset_dir, 'competition_test.csv'), os.path.join(dataset_dir, 'competition_val.csv')]
+        
+        annotations = []
+        for anno_file in anno_files:
+          with open(anno_file, 'r') as file:
+              reader = csv.reader(file)
```

Comment:
It may be more convenient to use pandas in the future, rather than having to actually read the file as a file object. See `pandas.read_csv`.